interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -266,7 +292,14 @@ private synchronized void tearDown() {
   private void run() {
     System.out.println("Running test " + testCase);
     try {
-      runTest(TestCases.fromString(testCase));
+      if (testCase.contains(",")) {
```

Comment:
There's no benefit to special-casing non-`,`-containing strings. The comma code can run in either case.

Reply:
Fixed.